cli/src/core/parse/profile.go

```diff
@@ -0,0 +1,48 @@
+package parse
+
+import (
+	"cli/core"
+	"cli/core/state"
+
+	"github.com/luno/jettison/errors"
+	"github.com/spf13/cobra"
+)
+
+func GetPackageSpecFromProfile(cmd *cobra.Command, config core.Config, packageSpec core.PackageSpec) (*core.PackageSpec, error) {
```

Comment:
The `packageSpec` arg does not seem to be used in the function. Can we change the signature of this function

Reply:
please check again, it's been used